From 0743da54540e5423ae63246df0cc2f074f0baef6 Mon Sep 17 00:00:00 2001
From: Erik Schumacher <erik.schumacher@iris-sensing.com>
Date: Thu, 26 Sep 2024 11:56:07 +0000
Subject: [PATCH] Modify imx93_matrixup_defconfig

- set target board
- use imx93-matrixup devicetree
- remove CONFIG_USB_TCPC
- remove CONFIG_VIDEO
---
 configs/imx93_matrixup_defconfig | 21 ++++-----------------
 1 file changed, 4 insertions(+), 17 deletions(-)

diff --git a/configs/imx93_matrixup_defconfig b/configs/imx93_matrixup_defconfig
index 540872d3048..203197a1332 100644
--- a/configs/imx93_matrixup_defconfig
+++ b/configs/imx93_matrixup_defconfig
@@ -10,11 +10,10 @@ CONFIG_SF_DEFAULT_SPEED=40000000
 CONFIG_ENV_SIZE=0x4000
 CONFIG_ENV_OFFSET=0x700000
 CONFIG_DM_GPIO=y
-CONFIG_DEFAULT_DEVICE_TREE="imx93-11x11-evk"
+CONFIG_DEFAULT_DEVICE_TREE="imx93-matrixup"
 CONFIG_SPL_TEXT_BASE=0x2049A000
 CONFIG_AHAB_BOOT=y
-CONFIG_USB_TCPC=y
-CONFIG_TARGET_IMX93_11X11_EVK=y
+CONFIG_TARGET_IMX93_MATRIXUP=y
 CONFIG_OF_LIBFDT_OVERLAY=y
 CONFIG_SYS_MONITOR_LEN=524288
 CONFIG_SPL_SERIAL=y
@@ -34,7 +33,7 @@ CONFIG_FIT_SIGNATURE=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_OF_SYSTEM_SETUP=y
 CONFIG_BOOTCOMMAND="run sr_ir_v2_cmd;run distro_bootcmd;run bsp_bootcmd"
-CONFIG_DEFAULT_FDT_FILE="imx93-11x11-evk.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx93-matrixup.dtb"
 CONFIG_SYS_CBSIZE=2048
 CONFIG_SYS_PBSIZE=2074
 CONFIG_ARCH_MISC_INIT=y
@@ -77,8 +76,8 @@ CONFIG_CMD_POWEROFF=y
 CONFIG_CMD_USB=y
 CONFIG_CMD_USB_MASS_STORAGE=y
 CONFIG_CMD_SNTP=y
-CONFIG_CMD_BMP=y
 CONFIG_CMD_CACHE=y
+CONFIG_CMD_CLS=y
 CONFIG_CMD_EFIDEBUG=y
 CONFIG_CMD_RTC=y
 CONFIG_CMD_TIME=y
@@ -179,18 +178,6 @@ CONFIG_USB_GADGET_VENDOR_NUM=0x1fc9
 CONFIG_USB_GADGET_PRODUCT_NUM=0x0152
 CONFIG_CI_UDC=y
 CONFIG_USB_PORT_AUTO=y
-CONFIG_VIDEO=y
-CONFIG_VIDEO_LOGO=y
-CONFIG_SYS_WHITE_ON_BLACK=y
-CONFIG_VIDEO_LCD_RAYDIUM_RM67191=y
-CONFIG_VIDEO_IMX_DW_DSI=y
-CONFIG_VIDEO_IMX_LCDIFV3=y
-CONFIG_SPLASH_SCREEN=y
-CONFIG_SPLASH_SCREEN_ALIGN=y
-CONFIG_BMP_16BPP=y
-CONFIG_BMP_24BPP=y
-CONFIG_BMP_32BPP=y
-CONFIG_VIDEO_ADV7535=y
 CONFIG_ULP_WATCHDOG=y
 CONFIG_SPL_RSA=y
 CONFIG_SHA384=y
-- 
2.46.0

